BedrockCore.h

```diff
@@ -36,6 +37,8 @@ class BedrockCore : public SQLiteCore {
     // the command hasn't timed out.
     bool isTimedOut(unique_ptr<BedrockCommand>& command);
 
+    RESULT prePeekCommand(unique_ptr<BedrockCommand>& command, bool exclusive = false);
```

Comment:
It crosses my mind that except for logging two different words, and calling the commands `prePeek` or `postProcess`, prePeekCommand and postProcessCommand are going to be identical. We should reuse code for both of them.

Reply:
They have a lot in common, but they aren't identical. `prePeekCommand` will return different values than `postProcessCommand`. `httpsRequests` can have different values between the two. Error handling will be different between the two...